### ISSUE TITLE
browser monitor: youtube videos without audio

### ISSUE THREAD

**Issue by ata-g-a** [subhra74/xdm#367]
![ss](https://user-images.githubusercontent.com/58735880/96348438-8f0c3900-10b5-11eb-9627-dd70f1e11a9c.PNG)

as you can see some of files show up without audio (null)
its not just this video and some of other videos have this issue which other than 720p others dont have audio

**Comment by Aryan-Gaurav** [comment]
Yes bro,today I came across the same problem.If you found any solution please help me.

**Comment by NKSHV** [comment]
for 1080p and above quality videos on youtube video and audio content come in separate streams
